assets/queries/ansible/aws/api_gateway_tracing/query.rego

```diff
@@ -0,0 +1,26 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  awsApiGateway := task["community.aws.aws_api_gateway"]
+  awsApiGateway.tracing_enabled == false
```

Comment:
should also check for "no", "false"
 isAnsibleFalse(answer) {
       lower(answer) == "no"
 } else {
       lower(answer) == "false"
 } else {
       answer == false
 }

Reply:
you are correct. I missed it. I've already changed